### PR TITLE
Add new bfe sachplan layer

### PR DESCRIPTION
**Pull request by daguer**

### CODE REVIEW

File: conf/uvek.conf.part [hunk 1]
```diff
@@ -1077,6 +1077,60 @@ source src_ch_astra_schwerverunfallte-kanton_pro_einwohner : def_searchable_feat
         from astra.unf_schwer
 }
 
+source src_ch_bfe_sachplan-geologie-tiefenlager_vernehmlassung_1 : def_searchable_features
+{
+    sql_db = uvek_dev
+    sql_query = \
+        SELECT facpt_id as id \
+            , 'feature' as origin \
+            , remove_accents(facptname_de)||' '||remove_accents(facptname_fr)||' '||remove_accents(facptname_it)||' '||remove_accents(fackind_de)||' '||remove_accents(fackind_fr)||' '||remove_accents(fackind_it)  as detail \
```
Comment (ltclm) [inline conf/uvek.conf.part]: please replace with 
```sql
remove_accents(concat_ws(' ', facptname_de, facptname_fr, facptname_it, facptname_de, fackind_de, fackind_fr, fackind_it)) as detail

the query above will fail if one of fields is NULL!
```

File: conf/uvek.conf.part [hunk 2]
```diff
@@ -1077,6 +1077,60 @@ source src_ch_astra_schwerverunfallte-kanton_pro_einwohner : def_searchable_feat
...
+        FROM bfe.geologische_tief_vernehm_fac_pt
+}
+
+source src_ch_bfe_sachplan-geologie-tiefenlager_vernehmlassung_2 : def_searchable_features
+{
+    sql_db = uvek_dev
+    sql_query = \
+        SELECT pmarea_id as id \
+            , 'feature' as origin \
+            , remove_accents(pmname_de)||' '||remove_accents(pmname_fr)||' '||remove_accents(pmname_it)||' '||remove_accents(pmkind_de)||' '||remove_accents(pmkind_fr)||' '||remove_accents(pmkind_it)  as detail \
```
Comment (ltclm) [inline conf/uvek.conf.part]: also replace with concat_ws function

File: conf/uvek.conf.part [hunk 3]
```diff
@@ -1077,6 +1077,60 @@ source src_ch_astra_schwerverunfallte-kanton_pro_einwohner : def_searchable_feat
...
+        FROM bfe.geologische_tief_vernehm_pm_area
+}
+
+source src_ch_bfe_sachplan-geologie-tiefenlager_vernehmlassung_3 : def_searchable_features
+{
+    sql_db = uvek_dev
+    sql_query = \
+        SELECT pmpt_id as id \
+            , 'feature' as origin \
+            , remove_accents(pmname_de)||' '||remove_accents(pmname_fr)||' '||remove_accents(pmname_it)||' '||remove_accents(pmkind_de)||' '||remove_accents(pmkind_fr)||' '||remove_accents(pmkind_it)  as detail \
```
Comment (ltclm) [inline conf/uvek.conf.part]: also replace with concat_ws function

**Comment by daguer** [comment]
@ltclm : concat_ws function used

**Pull request merged**